tktermwidget/tkterm.py

```diff
@@ -104,21 +103,22 @@ def __init__(self, master: Misc, autohide: bool = True, *args, **kwargs):
         self.index = 1
         self.current_process: Popen | None = None
 
-        # Bind events
+        # Bind events & tags
         self.text.bind("<Up>", self.up, add=True)
         self.text.bind("<Down>", self.down, add=True)
         self.text.bind("<Left>", self.left, add=True)
         self.text.bind("<Return>", self.loop, add=True)
         self.text.bind("<BackSpace>", self.left, add=True)
-        
+        self.text.mark_set("io", "end-1c")
+
         # TODO: Refactor the way we get output from subprocess
         self.text.bind("<Control-KeyPress-c>", self.kill, add=True) # Isn't working
 
         # History recorder
         self.history = open(HISTORY_PATH / "history.txt", "r+")
         self.historys = [i.strip() for i in self.history.readlines() if i.strip()]
         self.hi = len(self.historys) - 1
-	
+
     def directory(self):
         """Insert the directory"""
         self.text.insert(
```

Comment:
I generally dislike one line methods like this but I'll let it slide because it does keep down the line count overall